app/src/main/java/ch/epfl/sdp/musiconnect/database/SimplifiedDbEntry.java

```diff
@@ -1,4 +1,10 @@
 package ch.epfl.sdp.musiconnect.database;
 
 public abstract class SimplifiedDbEntry {
+    enum Fields {
+        username, firstName, lastName, email, typeOfUser, birthday, joinDate, location, events,
+        instruments, instrument, level,
+        host, participants, address, eventName, dateTime, description, visible,
+        leader, bandName, members
+    }
 }
```

Comment:
That's great! I will try to do the same for the intents' extra names, it will be more practical than writing the name